### PR TITLE
feat(MembershipsSDKAdapter): return membership status and all members

### PR DESCRIPTION
**Pull request by maloun96**

### CODE REVIEW

File: src/MembershipsSDKAdapter.js
Comment (lalli-flores) [inline src/MembershipsSDKAdapter.js:19]: Since you're removing the active members return, have you verified that this change does not break the "waiting for others..." state in the meeting component?

Comment (cipak) [reply src/MembershipsSDKAdapter.js:19]: good catch, thank you

Comment (cipak) [reply src/MembershipsSDKAdapter.js:19]: it's now fixed here https://github.com/webex/components/pull/165/commits/78e1f2699f71d1d8d087a565c6489e19718fed71#diff-eb5454f98c186e1cdd3067955c91f6ab2a64b7eee2de9026c49ee64901cf5f97R39

Comment (maloun96) [reply src/MembershipsSDKAdapter.js:19]: it's here https://github.com/webex/components/pull/165/files#diff-eb5454f98c186e1cdd3067955c91f6ab2a64b7eee2de9026c49ee64901cf5f97R39

Comment (lalli-flores) [reply src/MembershipsSDKAdapter.js:19]: Perfect!

**Review by lalli-flores — APPROVED** [review]

**Comment by lalli-flores** [comment]
@maloun96 I merged this PR and didn't realize that the checks didn't run. After merging CI failed because tests are not passing. Please open a PR to fix the broken tests so that the previous changes can be released.
All other PRs are blocked until this is fixed

**Pull request merged**
